eng/common/TestResources/README.md

```diff
@@ -72,6 +72,11 @@ setx AZURE_SEARCH_STORAGE_NAME $env:AZURE_SEARCH_STORAGE_NAME
 setx AZURE_SEARCH_STORAGE_KEY $env:AZURE_SEARCH_STORAGE_KEY
 ```
 
+Alternatively, when running the `New-TestResources.ps1` command, specify the `-OutFile` switch 
```

Comment:
NOTE: Only .net repo supports reading environment from file atm.

Wonder if we can change wording a bit so OutFile reads as a recommended way to do it for .net.

Reply:
implemented here - https://github.com/Azure/azure-sdk-tools/pull/690